modules/post/android/manage/remove_lock_root.rb

```diff
@@ -0,0 +1,41 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+
+class Metasploit4 < Msf::Post
+
+  include Msf::Post::Common
+
+  def initialize(info={})
+    super( update_info( info, {
+        'Name'          => "Android Root Remove Device Locks",
+        'Description'   => %q{
+            This module uses root privileges to remove the device lock.
+            In some cases the original lock method will still be present but any key/gesture will
+            unlock the device.
+        },
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'timwr' ],
+        'SessionTypes'  => [ 'meterpreter', 'shell' ],
+        'Platform'      => 'android',
```

Comment:
@timwr , you probably want to add `'Privileged'  => true` here since this requires root.